hs2-launcher/src/main/resources/hive-site.xml

```diff
@@ -60,21 +61,66 @@ used to store files.
         <name>hive.llap.io.enabled</name>
         <value>false</value>
     </property>
-    <!-- Use in memory derby DB for the metastore -->
+    <!-- END: Use Tez in local mode without LLAP -->
     <property>
         <name>javax.jdo.option.ConnectionURL</name>
         <value>jdbc:derby:memory:metastore;create=true</value>
+        <description>Use in memory derby DB as metastore</description>
     </property>
     <property>
         <name>datanucleus.schema.autoCreateAll</name>
         <value>true</value>
+        <description>Creates necessary schema on a startup if one doesn't exist</description>
     </property>
     <property>
         <name>hive.metastore.schema.verification</name>
         <value>false</value>
+        <description>If true, it disables autoCreateAll</description>
     </property>
     <property>
         <name>hive.tez.container.size</name>
         <value>1024</value>
     </property>
+    <!-- START: Create managed/transactional tables -->
+    <property>
+        <name>hive.create.as.acid</name>
+        <value>true</value>
+    </property>
+    <property>
+        <name>hive.default.fileformat</name>
+        <value>ORC</value>
+    </property>
+    <property>
+        <name>hive.default.fileformat.managed</name>
+        <value>ORC</value>
+    </property>
+    <property>
+        <name>hive.support.concurrency</name>
+        <value>true</value>
+    </property>
+    <property>
+        <name>hive.enforce.bucketing</name>
+        <value>true</value>
+    </property>
+    <property>
+        <name>hive.exec.dynamic.partition.mode</name>
+        <value>nonstrict</value>
+    </property>
+    <property>
+        <name>hive.compactor.initiator.on</name>
+        <value>true</value>
+    </property>
+    <property>
+        <name>hive.txn.manager</name>
+        <value>org.apache.hadoop.hive.ql.lockmgr.DbTxnManager</value>
+    </property>
+    <property>
+        <name>hive.compactor.worker.threads</name>
+        <value>1</value>
+    </property>
+    <property>
+        <name>hive.in.test</name>
+        <value>true</value>
```

Comment:
Out of curiosity did you understood why it is necessary to set this for ACID?

Reply:
Not 100%, what I was investigating right before finding this property was if ACID locking necessarily required to set up Zookeeper. It's nowhere in the doc, but it's stated here: [hive-site.xml](https://github.com/apache/hive/blob/master/data/conf/hive-site.xml#L186)

My best bet is that the property enables some kind of "local mode" at different places. I think it's safe to use since it's all over hive tests config files.